benchmarks/floop14.cpp

```diff
@@ -14,12 +14,12 @@
 #include <valarray>
 #endif
 
-BZ_NAMESPACE(blitz)
+namespace blitz {
 extern void sink();
-BZ_NAMESPACE_END
+}
```

Comment:
Can you change this (and all other BZ_NAMESPACE_END) lines to:
```
} // namespace blitz
```

Reply:
> Can you change this (and all other BZ_NAMESPACE_END) lines to:
> 
> ```
> } // namespace blitz
> ```

I tend to be an anti-fan of such non-maintainable comments. I guess most editors/IDEs make such information easily available (at least vim does ;))

Reply:
I tend to be anti-fan of that too... Note also two things:
1) Doing it now is difficult and would require manual editing of all the files as the pattern } is much less informative than BZ_NAMESPACE_END.
2) Even if I had replaced BZ_NAMESPACE_END with "} //  namespace blitz" when doing the work, it would have been wrong as BZ_NAMESPACE_END is used to close all types of namespaces (random, tensor, ...).

Consequently doing the request is and would have been quite a lot of work....